src/options/settings/components/container.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import SearchInjection from './SearchInjection'
+import { getLocalStorage, setLocalStorage } from 'src/search-injection/utils'
+import { SEARCH_INJECTION_KEY } from 'src/search-injection/constants'
+
+class Container extends React.Component {
+    constructor(props) {
+        super(props)
+        this.toggle = this.toggle.bind(this)
+    }
+
+    state = {
+        checked: false,
+    }
+
+    async componentDidMount() {
+        const checked = await getLocalStorage(SEARCH_INJECTION_KEY, true)
```

Comment:
Would name this to something more descriptive. Deeper into the code I see a checkbox is checked, but it would be nice if I could see what it represents immediately, like userDidEnableResults (or probably something better.)